vendor/k8s.io/kubernetes/pkg/kubeapiserver/options/plugins.go

```diff
@@ -138,6 +138,7 @@ func DefaultOffAdmissionPlugins() sets.String {
 		validatingwebhook.PluginName,            //ValidatingAdmissionWebhook
 		resourcequota.PluginName,                //ResourceQuota
 		storageobjectinuseprotection.PluginName, //StorageObjectInUseProtection
+		noderestriction.PluginName,              //NodeRestriction
```

Comment:
why to be different from the plain k8s here?
i'm afraid that it can confuse users. (at least i got confused a bit :-)

Reply:
what do you mean? can you ellaborate on what you mean by "different from plain k8s"

Reply:
@yamt answered here: https://github.com/rancher/k3s/pull/793#issuecomment-528507429